index.js

```diff
@@ -82,29 +82,42 @@ function toggleSound(){
 
     const flippedCards = []
     let matched=0;
-    // set image src when either card flip or show all cards
+    // set image src once at start of game
     const setImgSource = (card, i) =>{
-        card.children[0].src=cardArray[i]; // Setting image source for flipped front face
+        card.children[0].src = cardArray[i]; // Setting image source for flipped front face
+        card.children[1].src = 'images/cover.png'; //Setting image source for back face
         card.children[0].alt=cardArray[i].split('/').slice(-1)[0].split('.').slice(0, -1).join('.'); // Setting image file name as alt text for flipped front face
-        card.children[1].style.display="none";
+
+        //Addition transition to the card
+        card.style.transition = "transform 1s";
     }
 
     // unset image src when either card flip or show all cards
-    const unsetImgSource = (card) =>{
-        card.children[0].src="#"; // Removing image src so that it isn't visible through HTML
-        card.children[0].alt="card front face"; // Removing image alt so that it isn't visible through HTML
-        card.children[1].style.display="block";
+    // const unsetImgSource = (card) =>{
+    //     card.children[0].src="#"; // Removing image src so that it isn't visible through HTML
+    //     card.children[0].alt="card front face"; // Removing image alt so that it isn't visible through HTML
+    //     card.children[1].style.display="block";
+    // }
+

```

Comment:
unsetImageSource is important for the proper working of the game, this function prevents a user from inspecting the image src and win

see issue #2 for more details